### ISSUE TITLE
Partial Wings problem

### ISSUE THREAD

**Issue by SJuliez** [MegaMek/megamek#4081]
0.49.12: 

When configuring damage in the lobby and marking off even a single crit of a PW, it no longer confers a heat dissipation bonus and seems to count as destroyed, at least according to the MechView. (See Mech.getHeatCapacity(). The code dealing with remaining PW crit slots is not reached when it is destroyed) According to TO:AUE p.105 a PW loses jump bonus with crits but can stay functional and the heat bonus does not depend on crits but only requires the PW to be "functional".

**Comment by kuronekochomusuke** [comment]
looks like the lobby damage editor sets it to destroyed, if there is any damage assigned.  would probably be hard to determine generically how many hits it takes to destroy it.  

Entity.damageSystem
![image](https://user-images.githubusercontent.com/116095479/210120309-ced1a60c-d8a5-4129-883a-db1ff753ea21.png)


it also applies a hit to each location if the component if split across multiple locations.   instead of just applying one hit to one of the locations.

**Comment by SJuliez** [comment]
Yes, and the relevant part in GameManager has special treatment for Mek Shields at least but not PWs. I am currently trying to get Super-Cooled Myomer functional in MM (currently construction-only) and was trying to see how similar gear is treated so as not to reinvent the wheel. Seems it mostly just isn't (no wheel present) so I thought I'd add this issue.
